src/test/java/nextstep/courses/domain/field/CoverImageTest.java

```diff
@@ -0,0 +1,46 @@
+package nextstep.courses.domain.field;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+class CoverImageTest {
+
+    @Test
```

Comment:
```suggestion
    @Test
    @DisplayName("이미지 최대 크기가 넘으면 예외가 발생")
```

 DisplayName으로 저는 일반화된 설명을 넣을것 같아요. 😄

Reply:
각 테스트에 설명 추가하였습니다!